pkg/aws/pca_test.go

```diff
@@ -325,7 +325,7 @@ func TestIdempotencyToken(t *testing.T) {
 					Namespace: "fake-namespace",
 				},
 			},
-			expected: "f331cbfd0cc6569f58c12c3dbb238a4f",
+			expected: "6c414adffafef3d366beabcfcd74e8fa5a724beb4d3aa61e9e6ccdf32693b2e1", // Updated SHA-256 hash
```

Comment:
Hi @mritunjaysharma394, it looks like this test failed because the expected value should be "63e69830270b95081942a3d85034fdc97bb9f07b481846ac0003b5e809fc0b28"

Reply:
Thanks for pointing out that, just fixed it